test/io/write_operational_domain.cpp

```
@@ -135,6 +136,96 @@ TEST_CASE("Write simple operational domain", "[write-operational-domain]")
+        std::ostringstream os{};
+        write_operational_domain(opdom, os);
+
+        std::istringstream is{os.str()};
+
+        std::size_t num_lines = 0;
+
+        for (std::string line{}; std::getline(is, line); ++num_lines)
+        {
+            CHECK(expected.contains(line));
+        }
+
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)